addons/html_editor/static/src/others/dynamic_placeholder_plugin.js

```diff
@@ -74,13 +74,18 @@ export class DynamicPlaceholderPlugin extends Plugin {
      * @param {string} chain
      * @param {string} defaultValue
      */
-    onValidate(chain, defaultValue) {
+    async onValidate(chain, defaultValue) {
         if (!chain) {
             return;
         }
 
+        const fields = await this.services.orm.call(this.defaultResModel, 'fields_get', [], {});
```

Comment:
we are just using the partner_id from the field so can't we just get that rather than fetching all the fields 
can you give it a try to move this ORM call to open
Thanks!! :smile: